src/services/car/__tests__/messageLead.test.ts

```diff
@@ -1,8 +1,11 @@
+/* eslint-disable jest/no-try-expect */
```

Comment:
Please not disable linter rules for the entire file, only do it for the relevant lines instead.